src/reducers/filter.js

```diff
@@ -0,0 +1,12 @@
+import { CHANGE_FILTER } from '../actions';
+
+const filter = (filter = '', action) => {
```

Comment:
Oops, I think you missed setting the `filter` to `All`.

Let's set the filter to ALL, i.e `filter = All`

Screenshot from pathwright;
![image](https://user-images.githubusercontent.com/31897434/128005892-6832842b-c8af-435f-9e62-6a6abf5bc948.png)